### ISSUE TITLE
Update version

### ISSUE THREAD

**Issue by alanhogan** [laullon/gitx#198]
I’m not sure why I still have the “latest” version, 0.8.4, despite it being a year old and there being pull requests accepted as recently as 13 days ago.

**Comment by jackinloadup** [comment]
Agree! Please release new version there a many bug fixes that hinder usability. People on Snow Leopard can't compile this project anymore due to recent versions of Xcode not being available for Snow Leopard anymore which is an issue unto itself.

**Comment by jaywilliams** [comment]
+1

I'm on Snow Leopard and therefore can't compile it myself either.

**Comment by alanhogan** [comment]
Guys, there’s actually an active fork that I switched to:

http://rowanj.github.com/gitx/  
https://github.com/rowanj/gitx

**Comment by jaywilliams** [comment]
That's fantastic! I wasn't aware of that fork.
